contributing/branches.rst

```diff
@@ -35,5 +35,7 @@ Stable
 Stable-X.x
 ==========
 
-* There will be several more stable branches because we'll support the last three major versions of Zammad
-* If current stable version is 1.2.0 then the name of the branch is stable-1.2 and there also would be stable-1.1 and stable-1.0
+* Each minor and major version will have a stable branch (stable-x.x).
+* We are supporting the current develop and stable branch for security fixes and minor bugfixes
```

Comment:
What exactly do we want to tell here? 
We're suddendly talking about `develop` branch even through we're talking about stable Branches...?

What's the goal to do that? 
I wouldn't mention it at all.

I believe you want to say something like this for line 39:

```
We're supporting the current stable branch for security fixes and minor bugfixes.
```

You'll see why I'm going that short for line 41.